tests/lib/rules/scoped-elements.js

```diff
@@ -132,14 +132,38 @@ class MyElement extends ScopedElementsMixin(LitElement) {
     // ❌ INVALID
     //------------------------------------------------------------------------------
     invalid: [
+      {
+        /** ❌ Not declared */
+        code: `
+class MyElement extends ScopedElementsMixin(LitElement) {
+render() { return html\`<my-el></my-el>\`}
+}
+    `,
+        errors: [{ message: 'Element not declared in static scopedElements' }]
+      },
+      {
+        /** ❌ Not declared */
+        code: `
+class MyElement extends ScopedElementsMixin(LitElement) {
+connectedCallback() {
+  super.connectedCallback();
+}
+render() { return html\`<my-el></my-el>\`}
+}
+    `,
+        errors: [{ message: 'Element not declared in static scopedElements' }]
+      },      
         {
-            /** ❌ Not declared */
             code: `
 class MyElement extends ScopedElementsMixin(LitElement) {
+  get scopedElements() {
```

Comment:
You're not checking for `static get scopedElements`, you're checking for `get scopedElements`, which in theory should be super edge-casey (I dont expect anybody to use `get scopedElements`, but someone might). Just goes to show how specific you have to be when writing linting plugins. Added a failing test to catch it